packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -1041,6 +1042,15 @@ export const QueryBuilderResultPanel = observer(
                       {format}
                     </MenuContentItem>
                   ))}
+                  <MenuContentItem
+                    className="query-builder__result__export__dropdown__menu__item"
+                    onClick={(): void =>
+                      resultState.setIsExportOtherModalOpened(true)
+                    }
+                    disabled={queryBuilderState.changeDetectionState.hasChanged}
+                  >
+                    Other
```

Comment:
Suffix `...` as this prompts user for further actions

```suggestion
                    View Query Usage...
```